cmd/misc/main.go

```diff
@@ -1628,68 +1628,134 @@ func reIndexBlocks(start uint64, end uint64, bt *db.Bigtable, client *rpc.Erigon
 		}
 		end = uint64(lastBlockFromBlocksTable)
 	}
-
-	errFields := map[string]interface{}{
-		"start": start,
-		"end":   end,
+	transformers, importENSChanges, err := getTransformers(transformerFlag, bt)
+	if err != nil {
+		utils.LogError(nil, err, 0)
+		return
 	}
-
-	batchSize := uint64(10000)
-	for from := start; from <= end; from += batchSize {
-		targetCount := batchSize
-		if from+targetCount >= end {
-			targetCount = end - from + 1
+	if importENSChanges {
+		if err := bt.ImportEnsUpdates(client.GetNativeClient(), math.MaxInt64); err != nil {
+			utils.LogError(err, "error importing ens from events", 0)
+			return
 		}
-		to := from + targetCount - 1
+	}
 
-		errFields["from"] = from
-		errFields["to"] = to
-		errFields["targetCount"] = targetCount
+	cache := freecache.NewCache(100 * 1024 * 1024) // 100 MB limit
```

Comment:
```suggestion
	cache := freecache.NewCache(100 * 1024 * 1024) // 100 MB limit
```
We're using this value in multiple places, can we define it as const and place it at the top of the file pls?

Reply:
I rather not since I don't know what it does.